ui/src/app/kvm/components/KVMMeter/KVMMeter.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+
+import Meter from "app/base/components/Meter";
+
+type Props = {
+  allocated: number;
+  free: number;
+  segmented?: boolean;
+  total: number;
+  unit: string;
+};
+
+const KVMMeter = ({
+  allocated,
+  free,
+  segmented = false,
+  total,
+  unit = "",
```

Comment:
This has a default value, but in the type it isn't optional...

Reply:
thanks, fixed!